demo/index.html

```diff
@@ -40,17 +40,12 @@
           Prism.highlightAll();
         });
     </script>
-    <script type="module" src="../index.js"></script>
+    <script type="module" src="../index.js" data-demo-script="true"></script>
     <script type="module">
-      import { registerComponent } from "../index.js"
+      import {registerComponent} from '../dist/auro-accordion__bundled.js';
       registerComponent('custom-accordion');
     </script>
-    <script src="./index.min.js"></script>
-    <script>
-      initaccordionIndexExamples();
-    </script>
```

Comment:
I don't want to rip this out here. We are going to want to add some examples of expanding/contracting the accordion programatically and the functional code for those examples will get called here.